packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -24,43 +33,47 @@ const trace = makeTracer('LiqMin');
 const start = async zcf => {
   const { amm } = zcf.getTerms();
 
-  /**
-   * @param {ZCFSeat} debtorSeat
-   * @param {{ debt: Amount<'nat'> }} options
-   */
-  const debtorHook = async (debtorSeat, { debt }) => {
-    const debtBrand = debt.brand;
-    const {
-      give: { In: amountIn },
-    } = debtorSeat.getProposal();
+  const makeDebtorInvitation = curryInvitation(
+    zcf,
+    'Liquidate',
```

Comment:
?

Reply:
This is was in ce264a4 to show an option for not defining a method name at all. I think it reads well and doesn't require more indentation.